### ISSUE TITLE
Implement Political Entity types

### ISSUE THREAD

**Issue by amaurymartiny** [chronoscio/backend#40]
Implement all types in Annex 1 in: https://docs.google.com/spreadsheets/d/1lYX3tjX7DY0fsSadBmtzd6OSG3DMXi-irEQA5H2J9eA/edit#gid=2108598426

**Comment by whirish** [comment]
Did we come to a conclusion regarding this? I agree with Ville in that we should keep the government types purposefully broad to avoid disputes and leave any specifics up to narratives. I'm going to add the proposal tag until we come to a clear conclusion.

**Comment by amaurymartiny** [comment]
Ok agreed.

**Comment by wtokumaru** [comment]
I think that we should err on the side of generality and have everything be narrative (besides dates and polygons) until we have a solid platform. Then we can gradually implement more detailed descriptions over time without polluting the data with redundant and/or controversial descriptors.

**Comment by whirish** [comment]
Constantly updating or adding greater specifics to such values would make maintaining a narrative a more difficult process than it would otherwise need to be. I believe leaving any room for controversy entirely up to narratives is the best approach we can take as administrators of such a platform.

**Comment by wtokumaru** [comment]
What I mean is that I think we should stick to the facts for now and then if we want to add tags for stuff like government type we should add them later and treat them more as narrative for now. We do want to have some foundation that we can easily add to though if we do have plans to extend to different types of models that reference the same underlying data to show different types of maps (like a religion map or an economic system map).

**Comment by whirish** [comment]
Others have said that we should leave this out entirely of the site's data and instead leave it solely up to the narratives, thoughts?

**Comment by wtokumaru** [comment]
Yes please. If we do bring it in for whatever reason later, after we have a solid and complete base, worry about it then.

**Comment by ataalik** [comment]
It is a bit of an undertaking to have this feature since it's going to be somewhat vague.
The main difference would be if we if we implement this having this data structured would be very valuable for the database but if we don't, our job regarding our actual website would be easier. 

I'm on the side of implementing it but using the field sparingly. For example only for the political entities that clearly have a type according to consensus. Medieval Europe is a great example.
